mw-webapp/src/component/table/Table.tsx

```diff
@@ -1,38 +1,140 @@
 import {useState, useEffect} from "react";
-import {useReactTable, getCoreRowModel, flexRender} from "@tanstack/react-table";
-import {Report} from "src/model/report/Report";
-import {ReportService} from "src/service/Report";
-import {columns} from "src/component/table/columns";
+// import {useReactTable, getCoreRowModel, flexRender} from "@tanstack/react-table";
+// import {Report} from "src/model/report/Report";
+import {ReportService} from "src/service/ReportTest";
+// import {columns} from "src/component/table/columns";
 import {Button} from "src/component/button/Button";
 import styles from "src/component/table/Table.module.scss";
+import {Editable} from "../editable/Editable";
+// import {DayReport} from "src/model/businessModel/DayReport";
+// import {DayReport as DayReportDTO} from "src/model/firebaseCollection/DayReport";
+import {UserService} from "src/service/UserService";
+import {User} from "src/model/firebaseCollection/User";
+import {WayService} from "src/service/WayService";
+import {Way} from "src/model/firebaseCollection/Way";
 
 export const Table = () => {
-  const [data, setData] = useState<Report[]>([]);
+  // const [data, setData] = useState<DayReport[]>([]);
+  const [users, setUsers] = useState<User[]>([]);
+  const [ways, setWays] = useState<Way[]>([]);
 
   useEffect(() => {
-    ReportService.onValueFromRealTimeDb(setData);
+    // ReportService.onValueFromRealTimeDb(setData);
+    UserService.onValueFromRealTimeDb(setUsers);
+    // WayService.onValueByIdFromRealTimeDb(setWays, "vvv");
+    WayService.onValueFromRealTimeDb(setWays);
     () => {
       //TODO
       // removeEventListener from db if needed (read about handling event listeners
       // in react use effect components (when and whyu you shoud remove them))
     };
   }, []);
 
-  const table = useReactTable({
-    data,
-    columns,
-    getCoreRowModel: getCoreRowModel(),
-  });
+  // const table = useReactTable({
+  //   data,
+  //   columns,
+  //   getCoreRowModel: getCoreRowModel(),
+  // });
+
+  console.log(users);
+  console.log(ways);
+
+  // State for the input
+  const [task, setTask] = useState("");
+
+  const renderWays = (waysArray: string[]) => {
+    return waysArray.map((way, index) => (
+      <li key={index}>
+        {way}
+      </li>
+    ));
+  };
+
+  const renderAllWays = () => {
```

Comment:
RendeAllWays and render ways looks similar,why do you need two helpers, why don't u use only render ways?

Reply:
Right. Deleted renderAllWays func